### ISSUE TITLE
Gallery on Shopify

### ISSUE THREAD

**Issue by rform** [nanostudio-org/nanogallery2#90]
Hello,

I'm tyring to use nanogallery (basic example) on Shopify platform and getting following errors:

jQuery.Deferred exception: jQuery(...).nanogallery2 is not a function @https://cdn.shopify.com/s/files/1/2932/8992/t/1/assets/jquery.nanogallery2.min.js?15389630848517936225:41:20645
l@https://cdn.shopify.com/s/files/1/2932/8992/t/1/assets/jquery-3.3.1.min.js?15389630848517936225:2:29373
a/</c<@https://cdn.shopify.com/s/files/1/2932/8992/t/1/assets/jquery-3.3.1.min.js?15389630848517936225:2:29677
 undefined
jquery-3.3.1.min.js:2:30920
TypeError: jQuery(...).nanogallery2 is not a function
----------
I'm running following code: 
```
<div id="nanogallery2" data-nanogallery2 = '{"thumbnailHeight":  150,"thumbnailWidth":   150, "itemsBaseURL":     "https://nanogallery2.nanostudio.org/samples/"}'>
    <a href="berlin1.jpg" data-ngThumb="berlin1_t.jpg"> Berlin 1 </a>
    <a href="berlin2.jpg" data-ngThumb="berlin2_t.jpg"> Berlin 2 </a>
    <a href="berlin3.jpg" data-ngThumb="berlin2_t.jpg"> Berlin 3 </a>
  </div>
```
Same code is working on local machine. Any suggestions?

Thanks,
Rain

**Comment by Kris-B** [comment]
can you check that jquery is loaded only once in your page?

**Comment by rform** [comment]
It seems to be loaded only once. When I do not include my version then it shows another error about missing Jquery. In any case it got it now working with Javascript description option. Gallery itself is build very nicely and helping a lot. Good work!

**Comment by Kris-B** [comment]
;-)
